primitives/blockchain/src/backend.rs

```diff
@@ -279,8 +279,8 @@ pub mod well_known_cache_keys {
 	/// A list of authorities.
 	pub const AUTHORITIES: Id = *b"auth";
 
-	/// Current Epoch data.
-	pub const EPOCH: Id = *b"epch";
+	/// Current Session data.
+	pub const SESSION: Id = *b"epch";
```

Comment:
Old key has been preserved

Reply:
Nit: maybe a comment about why it's `epch`?